backend/lightning/lightning.go

```diff
@@ -117,11 +117,11 @@ func (lightning *Lightning) Activate() error {
 	lightningConfig.Accounts = append(lightningConfig.Accounts, &lightningAccount)
 
 	if err = lightning.setLightningConfig(lightningConfig); err != nil {
-		return err
+		return handleLightningSetupError(lightning, err)
```

Comment:
If `setLightningConfig` fails, calling `handleLightningSetupError` doesn't do anything, right? 
I think we can skip it here and move the content of `handleLightningSetupError` directly inside the if that checks `lightning.connect(true)` err value.

Reply:
I agree. I've moved the `handleLightningSetupError` logic only to `lightning.connect(true)` if
@Beerosagos PTAL